### PR TITLE
[Handbook] Update to GitHub Organizations and Repositories

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

This PR adds missing details for GitHub Organizations and Repositories.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c2153cc653e14fba2d4a10a220b0f66a1833160b at: https://61cb5fd96b639f94fdc31b8b--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5c712e4c1b54e0d4668998cf18a278b88ca3811b at: https://61cb65f0add95e141746c3dc--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b74e6443d8e03b3d4b53a1aed6e5576895d357d4 at: https://61cb6c746f54ffc0622aa732--layer5ng.netlify.app

File: src/components/handbook-navigation/content.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const content = [
   { id: 4, link: "/community/handbook/repository-overview", text: "Repository Overview" },
   { id: 5, link: "/community/handbook/projects", text: "Projects" },
   { id: 6, link: "/community/handbook/mentorship-programs", text: "Mentorship Programs" },
-  { id: 7, link: "/community/handbook/learn-layer5", text: "Learn Layer5" },
+  { id: 7, link: "/community/handbook/learn-layer5", text: "Learning" },
```
Comment (warunicorn19) [inline src/components/handbook-navigation/content.js:9]: Link need to be changed here.

Comment (warunicorn19) [reply src/components/handbook-navigation/content.js:9]: In contributor-journey

File: src/sections/Community/Handbook/index.js [hunk 2]
```diff
@@ -30,8 +30,8 @@ const handbookHome= () => {
         />
 
         <HandbookCard 
-          title="Contributor Journey" 
-          description="A leader is someone who can contribute to the Layer5 Community's growth by being accountable, participating in decision-making, and feeling responsible." 
+          title="Community Roles" 
+          description="Many roles exist in the community. Individuals who perform a role, bolster community and project growth by participating in decision-making and being accountable for those decisions." 
           to="/community/handbook/contributor-journey"
```
Comment (warunicorn19) [inline src/sections/Community/Handbook/index.js:35]:
```suggestion
          to="/community/handbook/community-roles" 
```

File: src/components/handbook-navigation/content.js (hunk 1 above)
Comment (leecalcote) [reply src/components/handbook-navigation/content.js:9]: Thank you. This approach to internal anchoring is untenable. It should be dynamic, not manual.

Comment (leecalcote) [reply src/components/handbook-navigation/content.js:9]:
```suggestion
  { id: 7, link: "/community/handbook/contributor-journey", text: "Learning" },
```

Comment (leecalcote) [reply src/components/handbook-navigation/content.js]: > Thank you. This approach to internal anchoring is untenable. It should be dynamic, not manual.

@YashKamboj something that you might look into?

**Comment by l5io** [comment]
🚀 Preview for commit 8b2e89713aed878e248c08e40f3f2baa717c0ef2 at: https://61cb88136b806506f1d74c41--layer5ng.netlify.app

File: src/components/handbook-navigation/content.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
 export const content = [
-  { id: 0, link: "/community/handbook/about", text: "About" },
+  { id: 0, link: "/community/handbook/contributor-journey", text: "About" },
   { id: 1, link: "/community/handbook/community", text: "Community" },
-  { id: 2, link: "/community/handbook/contributor-journey", text: "Contributor Journey" },
+  { id: 2, link: "/community/handbook/community-roles", text: "Contributor Journey" },
```
Comment (debo19) [inline src/components/handbook-navigation/content.js:4]:
```suggestion
  { id: 2, link: "/community/handbook/community-roles", text: "Community Roles" },
```

File: src/components/handbook-navigation/content.js [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
   { id: 3, link: "/community/handbook/contribution", text: "Contribution" },
   { id: 4, link: "/community/handbook/repository-overview", text: "Repository Overview" },
   { id: 5, link: "/community/handbook/projects", text: "Projects" },
   { id: 6, link: "/community/handbook/mentorship-programs", text: "Mentorship Programs" },
-  { id: 7, link: "/community/handbook/learn-layer5", text: "Learn Layer5" },
+  { id: 7, link: "/community/handbook/contributor-journey", text: "Learning" },
```
Comment (debo19) [inline src/components/handbook-navigation/content.js:9]:
```suggestion
  { id: 7, link: "/community/handbook/learn-layer5", text: "Learning" },
```

File: src/sections/Community/Handbook/repository.js [hunk 5]
```diff
@@ -30,6 +32,7 @@ const Repository = () => {
     <HandbookWrapper>
       <div className="page-header-section">
         <h1>Repository Overview</h1>
+        <p>A brief introduction to all of the Github repositories of Layer5 organization is offered below. This overview serves as a resource to newcomers seeking a perspective of the collective community efforts (from a code-centric perspective). See <a href="https://docs.google.com/document/u/0/d/1FzX-C_xy9hZ3Eu9dcCE0unhFV5LRUrb5YLn_MGYuG6Y/edit">Layer5 GitHub Process</a> for more related information.</p>
```
Comment (debo19) [inline src/sections/Community/Handbook/repository.js:35]: SharedScreenshot
This paragraph should go inside `<div className="content"></div>`

Comment (leecalcote) [reply src/sections/Community/Handbook/repository.js:35]: Thank you, @debo19!

**Comment by l5io** [comment]
🚀 Preview for commit a888b9f23e3daef172cb71bc1556e967b8a24f2d at: https://61cc676887cc53abf5ec3188--layer5ng.netlify.app

**Comment by YashKamboj** [comment]
I left a review , but forgot to submit it 😅, all resolved now.

**Review by YashKamboj — APPROVED** [review]

**Comment by leecalcote** [comment]
@YashKamboj, ha! There is one comment for you to give separate consideration to - https://github.com/layer5io/layer5/pull/2434#discussion_r776081254

**Comment by YashKamboj** [comment]
@leecalcote yeah, i saw, i am trying it.

**Pull request merged**